concordium-consensus/src/Concordium/KonsensusV1/Monad.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE DerivingVia #-}
```

Comment:
This file is very badly named. "Monad" explains nothing here.

Reply:
Agree it's not optimal. I'll see if I can come up with something better!